src/config/ConfigService.js

```diff
@@ -18,23 +18,68 @@ export default class ConfigService extends Service {
 
 	initialize() {
 		// Initialization of dynamic configuration
-		const headerLogoFull = document.getElementsByName('header-logo-full')[0]?.content;
-		const headerLogoMini = document.getElementsByName('header-logo-minimized')[0]?.content;
+		const headerLogoFullLight = document.getElementsByName('header-logo-full-light')[0]?.content;
+		const headerLogoMiniLight = document.getElementsByName('header-logo-minimized-light')[0]?.content;
+		const headerLogoFullDark = document.getElementsByName('header-logo-full-dark')[0]?.content;
+		const headerLogoMiniDark = document.getElementsByName('header-logo-minimized-dark')[0]?.content;
 		const title = document.getElementsByName('title')[0]?.content;
 		const customCSS = document.getElementsByName('custom-css-file')[0]?.content;
 
 		let dynamicConfig = {};
-		let brandImage = {};
-		// Add custom header full logo
-		if ((headerLogoFull != undefined) && (headerLogoFull != "")) {
-			brandImage["full"] = headerLogoFull;
-			dynamicConfig["brand_image"] = brandImage;
+
+		// Add custom header full light logo
+		if ((headerLogoFullLight != undefined) && (headerLogoFullLight != "")) {
+			dynamicConfig.brandImage = {light: {full: headerLogoFullLight}};
```

Comment:
@petrKavulok I would rather use

```
dynamicConfig["brandImage"] = {"light": {"full": headerLogoFull}};
```

Reply:
@petrKavulok I would do it even different way and use Object.assign which should help you to append a nested objects easily: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign